src/config.yaml

```diff
@@ -1,4 +1,9 @@
 options:
+  source:
+    default: ppa:canonical-infoblox/stable
```

Comment:
OK so you need to consider a OpenStack series aligned PPA schema here - networking-infoblox is an openstack project and has published releases since Mitaka - this is what I was talking about earlier in the week with regards to series alignment.

Reply:
Ok, James, sorry for that. I added a new pull request.